go.mod

```diff
@@ -12,6 +12,8 @@ require (
 	github.com/labstack/echo/v4 v4.1.16
 	github.com/labstack/gommon v0.3.0
 	github.com/leodido/go-urn v1.2.0 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
Are they used anywhere?

Reply:
They got automatically added when I followed your steps to add assets and compile. I am not fluent enough in Go to know if that's important. :/

Reply:
As far as I can see it is not used anywhere so you can remove them :)